Zend/zend_compile.c

```diff
@@ -3950,6 +3950,11 @@ static bool zend_compile_call_common(znode *result, zend_ast *args_ast, const ze
 			zend_error_noreturn(E_COMPILE_ERROR, "Cannot create Closure for new expression");
 		}
 
+		zend_ast_list *args = zend_ast_get_list(((zend_ast_fcc*)args_ast)->args);
+		if (args->children != 1 || args->child[0]->attr != _IS_PLACEHOLDER_VARIADIC) {
+			zend_error_noreturn(E_COMPILE_ERROR, "Cannot create a Closure for call expression with more than one arguments, or non-variadic placeholders");
```

Comment:
The "Cannot create a Closure" part is copied from "Cannot create Closure for new expression" above